README.md

```diff
@@ -165,6 +165,7 @@ The component accepts the following props:
 |**`sort`**|boolean|true|Enable/disable sort on all columns
 |**`filter`**|boolean|true|Show/hide filter icon from toolbar
 |**`search`**|boolean|true|Show/hide search icon from toolbar
+|**`showSearch`**|boolean|false|Initially displays search bar  
```

Comment:
What do you think about calling the option `searchOpen` instead? My concern is that I want to distinguish it from the numerous search options that exist now, and I worry it may cause confusion with the option that shows/hides the icon form the toolbar.